acceptance/c_test.go

```diff
@@ -0,0 +1,86 @@
+// Copyright 2016 The Cockroach Authors.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//	 http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or
+// implied. See the License for the specific language governing
+// permissions and limitations under the License.
+//
+// Author: Matt Jibson (mjibson@cockroachlabs.com)
+
+// +build acceptance
+
+package acceptance
+
+import "testing"
+
+// TestC connects to a cluster with C.
+func TestC(t *testing.T) {
+	testDocker(t, "c", []string{"/bin/sh", "-c", c})
+}
+
+const c = `
+set -e
+cat > main.c << 'EOF'
+#include <stdio.h>
+#include <stdlib.h>
+#include <string.h>
+#include <sys/types.h>
+#include <libpq-fe.h>
+
+int
+main(int argc, char **argv)
```

Comment:
In #3515 I suggested that we may want to go ahead and make a separate repo for each language instead of putting them all in the main repo. We're eventually going to want more complex tests (e.g. a full django or rails app, not just a trivial use of the driver) than would be reasonable to include in the main repo.